src/main/java/nl/hsac/fitnesse/util/BsnUtil.java

```diff
@@ -23,7 +23,7 @@ public String generateBsn() {
         int Nr2 = randomUtil.random(10);
         int Nr1 = 0;
         int SofiNr = 0;
-        if ((Nr9 == 0) && (Nr8 == 0) && (Nr7 == 0)) {
+        if (Nr9 == 0 && Nr8 == 0) {
```

Comment:
Why did you remove the third clause here (I have to admit I don't actually know this generation code very well, but you don't mention this in your PR text).

Reply:
@pvreeuwijk was this removal intentional? Can you explain to me why this clause should not (or does not have to) be there?